posthog/migrations/0241_subscription.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.13 on 2022-06-09 06:51
+
+from django.conf import settings
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Note this needs to be in a separate PR - otherwise we are not able to roll this one back.

Reply:
The issue is that these PRs are not practically revertable (we'd get into a situation where deploys don't occur) so you need to be quite sure it's safe. 56 files isn't that :sweat_smile: 

I remembered this being documented in the handbook but couldn't find it. Here's a related incident though: https://github.com/PostHog/postmortem/blob/master/2022-02-18-rename-tags-column.md